packages/react/src/useTableUtils/helpers.ts

```diff
@@ -171,6 +195,10 @@ export const getTableColumns = (spec: TableSpec) => {
         }
       }
 
+      if (field.fieldType === GadgetFieldType.DateTime) {
+        column.includeTime = (field.configuration as GadgetDateTimeConfig).includeTime;
+      }
```

Comment:
Another way you could do is with this:
```ts
      if (field.configuration.__typename === 'GadgetDateTimeConfig') {
        column.includeTime = field.configuration.includeTime;
      }
```

That way you don't need to cast the config type.

Reply:
Good call. Updated